### ISSUE TITLE
Remembering "Query Log" in "Compact" mode in the next session

### ISSUE THREAD

**Issue by binarweb** [AdguardTeam/AdGuardHome#4704]
### Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * v0.107.7
* **How did you install AdGuard Home:**
  * GitHub releases
* **How did you setup DNS configuration:**
  * Router
* **If it's a router or IoT, please write device model:**
  * Raspberry Pi 3 Model B
* **CPU architecture:**
  * ARM
* **Operating system and version:**
  * Raspbian GNU/Linux 10 (buster)

Filtering query logs in compact mode is useful and it works great.
However, the next time I check its web interface, the browser reloads and the "Compact" mode switches back to "Default" mode.

### Expected Behavior

The "Compact" mode should remain active between page reloads.

</details>

**Comment by AdamAnon** [comment]
Yes, thanks! This would be very nice if Compact mode remained across sessions and page reloads.
